lib/flappy_search_bar.dart

```diff
@@ -200,6 +222,24 @@ class SearchBar<T> extends StatefulWidget {
   /// Set a padding on the list
   final EdgeInsetsGeometry listPadding;
 
+  /// Set a padding on all the search content
+  final EdgeInsetsGeometry contentPadding;
+
+  /// Set a leading widget before the search bar
+  final Widget leading;
+
+  /// Set a trailing widget after the search bar
+  final Widget trailing;
+
+  /// Whether the query bar should be focused on build
+  final bool autoFocus;
+
+  /// The focus node to be used in the search widget
+  final FocusNode focusNode;
+
+  /// The text editing controller to be used in the search widget
+  final TextEditingController textEditingController;
```

Comment:
Why dont you use the Search Bar Controller as an interface to edit the TextField ?
Using a "Controller" in a package should enable the user to control anything, don't you agree ?